### PR TITLE
remove krun dependence on parser code

### PR DESCRIPTION
**Pull request by dwightguth**
@radumereuta @bmmoore please review

### CODE REVIEW

**Comment by dwightguth** [comment]
@radumereuta @bmmoore please review

File: src/javasources/KTool/src/org/kframework/kil/Variable.java [hunk 1]
```diff
@@ -54,7 +54,6 @@ public Variable(String name, Sort sort, boolean freshVariable, boolean freshCons
         this.name = name;
         this.freshVariable = freshVariable;
         this.freshConstant = freshConstant;
-        this.userTyped = true;
```
Comment (radumereuta) [inline src/javasources/KTool/src/org/kframework/kil/Variable.java:57]: Explain this please.

Comment (dwightguth) [inline src/javasources/KTool/src/org/kframework/kil/Variable.java:57]: I put it in a while back, but it isn't really appropriate. It was just a mistaken fix for a bug that was better fixed another way. Brandon asked me to clean it up, so I did with this diff.

**Comment by radumereuta** [comment]
Ok, then if Brandon agrees too, lgtm.

File: src/javasources/KTool/src/org/kframework/kil/Variable.java (hunk 1 above)
Comment (bmmoore) [inline src/javasources/KTool/src/org/kframework/kil/Variable.java:57]: This is the only thing I wanted/needed from this fix, so as long as tests still pass I'm fine with this pull request.

**Pull request merged**